akka/src/main/scala/cakesolutions/kafka/akka/Offsets.scala

```diff
@@ -26,25 +38,50 @@ case class Offsets(offsetsMap: Map[TopicPartition, Long]) extends AnyVal {
   def toCommitMap: Map[TopicPartition, OffsetAndMetadata] =
     offsetsMap.mapValues(offset => new OffsetAndMetadata(offset))
 
+  /**
+    * Keep only the offsets for partitions that are also in the given set of partitions.
+    *
+    * @param tps partitions to filter the offsets with
+    * @return filtered offsets
+    */
   def keepOnly(tps: Set[TopicPartition]): Offsets =
     copy(offsetsMap.filter { case (t, _) => tps(t) })
 
+  /**
+    * Remove the given partitions from the offsets.
+    *
+    * @param tps partitions to remove
+    * @return filtered offsets
+    */
   def remove(tps: Set[TopicPartition]): Offsets =
     copy(offsetsMap -- tps)
 
+  /**
+    * Does the offsets contain any offset information.
```

Comment:
This doesn't read so well. It's not much better, but how about something like "Whether there are no offsets."?